citi-onboarding/Aqua-Jr
# Bugfix/footer css

## What I did
- Fixed the bugs on the Footer and Post To Blog sections.
## How to test
- On one terminal you should type the following commands to access the client folder and run yarn start
```
    cd client
    yarn start
```
- If the previous step was followed correctly you should now have a blank localhost300 page open on your server
- Moving on, go to your second terminal and access the server folder, where you should run either **node index.js** or **nodemon index.js**
```
    cd server
    nodemon index.js
```
- Now you're all set and should be able to view the website on your localhost3000 page. To properly test it, put it through the following test: change your screen size both randomly and following the sizes suggested by the server

- If both the web and mobile versions work just as shown in the mockup, everything's fine :)
